src/context/todoContext/todoContext.tsx

```diff
@@ -0,0 +1,26 @@
+import { createContext } from 'react';
+import { Todo } from '../../types/Todo';
+
+export interface TodoContextProps {
+  todos: Todo[],
+  size: number,
+  countCompleted: number,
+  setTodos: (todos: Todo[]) => void,
+  addTodo: (todo: Todo) => void;
```

Comment:
Be careful, because after defining the type, there should not be `;`.

Reply:
fix Roman's comment